packages/ui/src/TapToEdit.tsx

```diff
@@ -110,6 +110,23 @@ export const TapToEdit = ({
       } else if (fieldProps?.type === "multiselect") {
         // ???
         displayValue = value.join(", ");
+      } else if (fieldProps?.type === "address") {
+        const city = value?.city
```

Comment:
Nice job on the logic with this! Could we remove the nested ternaries to increase readability?

```
let city = '';
if(value?.city){
  city = value?.state || value.zipcode ? `${value.city}, ` : `${value.city}`
}
```
I think that's equivalent...

Reply:
good call, done!